docs/book/how-to/stack-deployment/register-a-cloud-stack.md

```diff
@@ -0,0 +1,130 @@
+---
+description: Seamlessly register a cloud stack using existing infrastructure
+---
+
```

Comment:
Same to add something like  this here: ![image](https://github.com/zenml-io/zenml/assets/4792082/1f930cdc-449e-4a9a-ba1a-02aaae85da91)

Reply:
Since this is not limited to deployed instances of ZenML, I won't add this for now.